gentle_gnomes/src/indicator.py

```diff
@@ -45,12 +49,10 @@ def __init__(self, name: str, city: City):
             x[i] = int(year)
             y[i] = values['avg']
 
-        self.rate = self._calc_slope(x, y)
+        self.rate = stats.linregress(x, y)[0]
 
-    @staticmethod
-    def _calc_slope(x: np.ndarray, y: np.ndarray) -> float:
-        slope, *_ = stats.linregress(x, y)
-        return slope
+        self.x = x.tolist()
+        self.y = y.tolist()
```

Comment:
Should these be converted to JSON? That would be the safest thing to do but in practice Python array's `repr` is compatible with JSON.

Reply:
Might as well!